### PR TITLE
updated pixel_shuffle in opset 11 to use depthToSpace

### PR DESCRIPTION
**Pull request by neginraoof**

### CODE REVIEW

File: torch/onnx/symbolic_opset11.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from torch.onnx.symbolic_helper import parse_args, _unimplemented
```
Comment (lara-hdr) [inline torch/onnx/symbolic_opset11.py]: Can you blacklist all the other ops that are updated in ONNX upset 11?

You'll need to create an array like 'black_listed_operators', and black list the elements in it; 
you can look at symboli_opset7.py, that does that for some ops in opset 7.

Comment (neginraoof) [reply torch/onnx/symbolic_opset11.py]: Thanks Lara, I added the list.

File: torch/onnx/symbolic_opset11.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+from torch.onnx.symbolic_helper import _black_list_in_opset
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 11
+
+black_listed_operators = [
+    "loop", "cumsum", "round", "bitshift", "equal"
```
Comment (lara-hdr) [inline torch/onnx/symbolic_opset11.py]: can you add ne as well? (it uses equal)
Make sure to add the symbolic names (I think the symbolic name for equal is "eq").

Comment (neginraoof) [reply torch/onnx/symbolic_opset11.py]: Thanks. There are no symbolics for bitshift, round, and loop

Comment (lara-hdr) [reply torch/onnx/symbolic_opset11.py]: in that case you can remove them and keep cumsum, eq and ne

**Comment by neginraoof** [comment]
@bddppq please review

**Review by lara-hdr — APPROVED** [review]
Looks good, thanks!

**Comment by neginraoof** [comment]
@bddppq please review and merge if possible.

**Review by bddppq — APPROVED** [review]
LGTM

**Pull request closed without merging**